base/generator.jl

```diff
@@ -20,6 +20,12 @@ function next(g::Generator, s)
     g.f(v), s2
 end
 
+"""
+    generate(f, c...) -> iterator
+
+Apply `f` to each element of `c`.
+For multiple collection arguments, apply f elementwise.
+"""
 generate(f, c...) = Generator(f, c...)
```

Comment:
please don't export this. It's a very small wrapper and unfortunately close in syntax to `@generated`